### PR TITLE
Adds stackoverflow and google links on debug mode

### PR DESCRIPTION
**Pull request by ipinak**
Links which point to StackOverflow search page or Google search page only when in debug mode.
Adds missing dependency from `CONTRIBUTING.rst` document.

### CODE REVIEW

**Comment by davidism** [comment]
Thanks for the patch, but I don't think this belongs in the debugger.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Werkzeug must be installed for all tests to pass::
 The minimal requirement for running the testsuite is ``py.test``.  You can
 install it with::
 
-    pip install pytest
+    pip install pytest mock
```
Comment (jeffwidman) [inline CONTRIBUTING.rst]: This is still probably valid and worth submitting as it's own PR

Comment (davidism) [inline CONTRIBUTING.rst]: `mock` isn't a requirement. `requests`, on the other hand, is.

Comment (davidism) [inline CONTRIBUTING.rst]: 16fdd66c0dc8cfd6e6ec6c90569ba4f667f8b536

**Pull request closed without merging**